pkg/container/allowed_base_image_check.go

```diff
@@ -71,17 +71,22 @@ func (a *AllowedBaseImage) Test() *txqualitychecks.QualityResult {
 		}
 	}
 
-	return &txqualitychecks.QualityResult{Passed: checkPassed, ErrorDescription: buildErrorDescription(deniedBaseImages)}
+	return &txqualitychecks.QualityResult{Passed: checkPassed, ErrorDescription: buildErrorDescription(deniedBaseImages, "cli"), ErrorDescriptionWeb: buildErrorDescription(deniedBaseImages, "web") }
 }
 
 func (a *AllowedBaseImage) IsOptional() bool {
 	return false
 }
 
-func buildErrorDescription(deniedImages []string) string {
+// Function to return error message of failed test.
+// There are two types of formatting: cli (default) and web
+func buildErrorDescription(deniedImages []string, errorFormat string) string {
```

Comment:
I think even if the supported options are defined as comment, there should be a way to let the compiler help choosing the right option. Which means we should implement a type similar to enums in other languages.
The last two code snippets from [this SO post](https://stackoverflow.com/a/14426447) could be a good approach.

Following this, we could also think about not having a parameter at all, but globally setting the desired logOutputFormat. In this case, any app using the quality check library could set the proper value, before running the checks.